_posts/2023-06-30-trino-fest-2023-apacheiceberg.md

```diff
@@ -57,6 +57,9 @@ may give you what you want from both patterns. With Iceberg, you have the choice
 different CDC patterns can be supported for you to adjust your usage to your
 specific needs. Check out the video and review the slides for more details!
 
+Want to read more about CDC? Check out some of Ryan Blue's blog posts at
+[Tabular](https://tabular.io/blog/)!
```

Comment:
Please link to the actual cdc series posts - https://tabular.io/blog/hello-world-of-cdc/
https://tabular.io/blog/cdc-data-gremlins/